README.md

```diff
@@ -1,20 +1,20 @@
-# Shotgun for Unity: the tk-unity engine
+# Shotgrid for Unity: the tk-unity engine
 
-This repository is part of the Shotgun for Unity system. This provides the
-Shotgun engine description to allow Shotgun to communicate with Unity.
+This repository is part of the Shotgrid for Unity system. This provides the
+Shotgrid engine description to allow Shotgrid to communicate with Unity.
 
-The full documentation is maintained as part of the [com.unity.integrations.shotgun](https://docs.unity3d.com/Packages/com.unity.integrations.shotgun@latest) package.
+The full documentation is maintained as part of the [com.unity.integrations.shotgrid](https://docs.unity3d.com/Packages/com.unity.integrations.shotgrid@latest) package.
```

Comment:
This URL will not work until we release I guess?
Can you open a Jira story to validate the URLs post-release please?

Reply:
@jibola1997 : Will you open a Jira issue?